### PR TITLE
Fix/l2 451 inconsistency in neuron icp amount overview

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Fix inconsistency in neurons:
- different info in proposer modal (under Locked...) (discussed with Misha)
- neuron ICPs
- same sorting as in Flutter (discussed with Misha)
https://dfinity.atlassian.net/browse/L2-451
image

# Changes

- updated ICP stake amount (`votingPower` for proposer version and `ICP get stake => cachedNeuronStake - neuronFees;` for general)
- show staked date instead of dissolve delay in proposer view
- add sortedNeuronStore

# Tests

- added test for sorting

# Screens

image

image

image

image

### CODE REVIEW

File: frontend/svelte/src/lib/components/neurons/NeuronCard.svelte [hunk 1]
```diff
@@ -23,10 +24,13 @@
   let isCommunityFund: boolean;
   $: isCommunityFund = hasJoinedCommunityFund(neuron);
   let neuronICP: ICP;
-  $: neuronICP =
-    neuron.fullNeuron?.cachedNeuronStake !== undefined
-      ? ICP.fromE8s(neuron.fullNeuron.cachedNeuronStake)
-      : ICP.fromE8s(BigInt(0));
+  $: neuronICP = proposerNeuron
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/neurons/NeuronCard.svelte:27]: should we put this in a function - i.e. not implement this logic within the component?

File: frontend/svelte/src/lib/components/neurons/NeuronCard.svelte [hunk 2]
```diff
@@ -23,10 +24,13 @@
...
+    ? ICP.fromE8s(neuron.votingPower)
+    : neuron.fullNeuron?.cachedNeuronStake !== undefined
+    ? ICP.fromE8s(
+        neuron.fullNeuron.cachedNeuronStake - neuron.fullNeuron.neuronFees
```
Comment (lmuntanerDfinity) [inline frontend/svelte/src/lib/components/neurons/NeuronCard.svelte:31]: Isn't this confusing? Did you talk with Mischa about improvements here? Can we have a comment on why one calculation in one and another in the other one?

File: frontend/svelte/src/lib/modals/proposals/ProposerModal.svelte [hunk 3]
```diff
@@ -34,7 +34,7 @@
 
   {#if neuron !== undefined}
     <div class="content">
-      <NeuronCard {neuron} />
+      <NeuronCard proposerNeuron {neuron} />
```
Comment (lmuntanerDfinity) [inline frontend/svelte/src/lib/modals/proposals/ProposerModal.svelte:37]: This differences in the NeuronCard are a thing that I would like to challenge from the design phase.

File: frontend/svelte/src/lib/stores/neurons.store.ts [hunk 4]
```diff
@@ -53,3 +56,8 @@ const initNeuronSelectStore = () => {
 };
 
 export const neuronSelectStore = initNeuronSelectStore();
+
+export const sortedNeuronStore: Readable<NeuronInfo[]> = derived(
```
Comment (lmuntanerDfinity) [inline frontend/svelte/src/lib/stores/neurons.store.ts:60]: 👍

**Review by lmuntanerDfinity — APPROVED** [review]
Just a few comments, but for now it's ok.

File: frontend/svelte/src/lib/components/neurons/NeuronCard.svelte (hunk 2 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/neurons/NeuronCard.svelte:31]: We discussed the Staked vs Delay in details on Friday. I'll talk to him regarding the ICP value.

File: frontend/svelte/src/lib/components/neurons/NeuronCard.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/neurons/NeuronCard.svelte]: Logic was simplified.

**Review by peterpeterparker — APPROVED** [review]
thx

**Pull request merged**